tox.ini

```diff
@@ -1,16 +1,23 @@
 [tox]
-envlist = lint, test
+envlist = flake8, pylint, test
 skipsdist=True
 
 [testenv]
 ignore_errors = False
 basepython = python3.6
 
-[testenv:lint]
+[testenv:flake8]
```

Comment:
We could put both `flake8` and `pylint` in the same `lint` step. I'm not saying we should. It's just something to consider. It's probably better to be able to run them independently.

Reply:
Yeah I wasn't sure which way to go.

Reply:
I think you made the right call. I'd like to be able to just `flake8` by itself.

Reply:
I've seen quite a few `tox.ini` that didn't even bother with environments like that. As in their `testenv` would run the commands `pytest`, `flake8` and `pylint`, like so:
```
[testenv]
ignore_errors = False
basepython = python3.6
use_develop = True
commands =
    flake8 --max-line-length 99 --max-complexity 12 nim
    - pylint nim
    pytest {posargs}
deps =
    flake8
    mock
    pylint
    pytest
    pytest-cov
```
Personally that would be fine for me because I'll almost certainly always run `tox` and never `tox -e something`, so I wouldn't mind losing the flexibility.
Maybe we should revisit that once more people have started using `tox` to see if we should keep the environments as they are or not.